### DISCUSSION TITLE
load page to play video and record, for a while the application will be crashed.

### DISCUSSION THREAD

**Discussion by swneng** [cefsharp/CefSharp#3977] [General]
Error message:

Exception Code: 0xe0000008
Error code offset: 0x0012b5b2
Error Application: CefSharp.BrowserSubprocess.exe

 Thanks

**Comment by amaitland** [comment]
Check the log file for errors see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

If there is nothing meaningful there then you'll need to get a stack trace see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#unmanaged-crashes-when-the-process-dies

If you require further assistance then please answer the questions listed at https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following
